site/content/en/docs/tasks/run_python_jobs.md

```diff
@@ -115,118 +59,15 @@ python install-kueue-queues.py
 You can also target a specific version:
 
 ```bash
-python install-kueue-queues.py --version 0.3.2
+python install-kueue-queues.py --version 0.4.0
```

Comment:
makes sense!

Reply:
The code has the version there too.

I dont know if we can replace that there. We might just have to update the python code example in the future ourselves.

Reply:
it will probably go stale.
We could remove the version from the code, but it should be fine if it goes stale.

Reply:
We could also have the default install the latest from the master/main branch.

Reply:
I usually am not the biggest fan of latest since we might not know what can break. (it shouldn't but just been burned too many time with docker :latest tag)

I am also fine with the example default getting stale.

Reply:
Using `:main` is fine for 2 reasons:
- it's just sample code, not meant to be used in production as-is.
- It's already parameterized, and we will keep the parameter up-to-date in this file using the variable.